.travis.yml

```diff
@@ -3,21 +3,24 @@ sudo: false
 dart:
   - dev
   - stable
+  - 1.23.0
```

Comment:
Are these really necessary? Seems like just dev/stable + lowest bound for environment in the pubspec would be sufficient.